_search-plugins/knn/jni-libraries.md

```diff
@@ -8,7 +8,10 @@ has_children: false
 
 # JNI libraries
 
-To integrate [*nmslib*'s](https://github.com/nmslib/nmslib/) and [*faiss*'s](https://github.com/facebookresearch/faiss/) Approximate k-NN functionality (implemented in C++) into the k-NN plugin (implemented in Java), we created a Java Native Interface, which lets the k-NN plugin make calls to the native libraries. We create 3 libraries: `libopensearchknn_nmslib`, the JNI library that interfaces with nmslib, `libopensearchknn_faiss`, the JNI library that interfaces with faiss, and `libopensearchknn_common`, a library containing common shared functionality between native libraries.
+To integrate [nmslib](https://github.com/nmslib/nmslib/) and [faiss](https://github.com/facebookresearch/faiss/) approximate k-NN functionality (implemented in C++) into the k-NN plugin (implemented in Java), we created a Java Native Interface, which lets the k-NN plugin make calls to the native libraries. We create 3 libraries: `libopensearchknn_nmslib`, the JNI library that interfaces with nmslib, `libopensearchknn_faiss`, the JNI library that interfaces with faiss, and `libopensearchknn_common`, a library containing common shared functionality between native libraries.
```

Comment:
Related to my previous comments, I think it would be better to use "you created" or "you use" instead of "we created" in both instances because it centers the actions the user is performing (we are not performing the actions with them).

Reply:
In this case, the description seems to be talking about an approach OpenSearch took to integrating libraries written in C++ with the k-NN plugin, written in Java. It's more of an explanation about how the plugin communicates with the two libraries. But it is a little wonky. I rewrote that second sentence, "The interface includes three libraries: ..."

Reply:
Ah ok, makes sense.